arrays/array_practice.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+
+#Create an array variable with the the numbers 1 through 10
+array = (1..10).to_a
+# 1...2...3...4...5...6...7...8...9...10...
+puts "#{array.join('...')}" + "..."
+# T-10, 9, 8, 7, 6, 5, 4, 3, 2, 1...  BLASTOFF!
+puts 'T-' + array.reverse.to_s.tr('[]',"") + '...  BLASTOFF!'
+# The last element is 10
+puts "The last element is #{array.last}"
+# The first element is 1
+puts "The first element is #{array.first}"
+# The third element is 3
+puts "The third element is #{array[2]}"
+# The element with an index of 3 is 4
+puts "The element with an index of #{array[2]} is #{array[3]}"
+# The second from last element is 9
+puts "The second from last element is #{array[8]}"
```

Comment:
Can you find a way to find the "second from last element" no matter how many elements are in the array?  Hint: Try negative #'s